### PR TITLE
[web] fixed screen height

### PR DESCRIPTION
**Pull request by oxyii**
ref [expo/expo#5081](https://github.com/expo/expo/issues/5081)

### CODE REVIEW

File: packages/webpack-config/web-default/index.html [hunk 1]
```diff
@@ -19,15 +19,18 @@
         -webkit-overflow-scrolling: touch;
         margin: 0px;
         padding: 0px;
-        /* Allows content to fill the viewport and go beyond the bottom */
-        min-height: 100%;
+        /* DO NOT Allows content to fill the viewport and go beyond the bottom */
```
Comment (EvanBacon) [inline packages/webpack-config/web-default/index.html]: Fix grammer

**Comment by EvanBacon** [comment]
This may cause major visual regressions, we should test against changes like this in the future.

File: packages/webpack-config/web-default/index.html (hunk 1 above)
Comment (oxyii) [reply packages/webpack-config/web-default/index.html]: @EvanBacon My english is bad. Please do it your-self.

**Comment by FiberJW** [comment]
This causes a lot of damage, so I'm going to revert this. Sorry for the confusion :(

**Pull request merged**